src/controllers/generalSettings.js

```diff
@@ -0,0 +1,21 @@
+const fixedVariables = require('../database/models/fixedVarible');
+
+exports.get = (req, res) => {
+  fixedVariables.findAll(
+    {
+      attributes:
+      ['dolar', ['exemption_resident', 'exemptionResident'],
+        ['tax_value', 'taxValue'], ['contribution_institute', 'contributionInstitute'],
+        ['contribution_employee', 'contributionEmployee'], ['saving_ratio', 'savingRatio']],
+    },
+  ).then((result) => {
+    res.render('generalSettings', { result, cssFile: 'generalSettings' });
+  });
+};
+
+exports.post = (req, res) => {
+  const input = req.body;
+  fixedVariables.update(input, { where: { id: 1 } }).then(() => {
+    res.redirect('/generalsetting');
```

Comment:
fix the route name:
``` js 
'/generalsetting' 
```

Reply:
Where is the problem with the route ? @ishak52

Reply:
use hyphen to write your routs @mohannadhanafi 
[for more info](https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)